src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -2836,27 +2836,6 @@ public bool IsPassive()
             return _engine.CurrentState == RachisState.Passive;
         }
 
-        public async Task<(long Index, object Result)> SendToLeaderAsync(CommandBase cmd)
```

Comment:
This is a leftover and I removed it just to clean here and make the code more clear